src/main/java/com/simplewebapp/service/UserService.java

```diff
@@ -9,14 +9,14 @@
  */
 public interface UserService {
 
-  public void addUser(User user);
+  void addUser(User user);
 
-  public void updateUser(User user);
+  void updateUser(User user);
 
-  public User getUser(int id);
+  User getUser(int id);
 
-  public void deleteUser(int id);
+  void deleteUser(int id);
```

Comment:
Another useful is to return boolean on delete. This way the Controller can identify if something happened while deleting. (For example if you try to delete a user and there's a Foreign Key constrain. The check should be done on Service layer but Controller needs to return something useful to the API (User). By returning false you know something happened (expected) and you can create the appropriate message for the user. If something unexpected happens (exception) then it will fail anyway.